docs/specs/event.md

```diff
@@ -479,6 +480,29 @@ Use this event to add custom fields to the JWT access token.
 
 - `identities`: This contain all Login ID identities, OAuth identities, or LDAP identities that the user has.
 
+#### oidc.id_token.pre_create
+
+Occurs right before the ID token is issued.
+Use this event to add custom fields to the ID token.
+
+```json5
+{
+  "payload": {
+    "user": { /* ... */ },
+    "identities": [ ],
+    "jwt": {
```

Comment:
This shuold be `id_token` according to https://github.com/authgear/authgear_deno_hook/pull/13/files